### DISCUSSION TITLE
Bagaimana cara mengatasi high variance saat akan melakukan PCA

### DISCUSSION THREAD

**Discussion by reyhanpurnawan** [teamalgoritma/community#491] [Q&A] [answered]
Selamat malam, saya kesulitan saat akan melakukan pca. pada saat perform dimensionality reduction dim 1 mengcapture lebih dari 95% of variance. bagaimana cara mengatasi hal tersebut ya? apakah tidak apa apa jika dibiarkan seperti itu. Terimakasih 
[Final Exam Intro Data Science.zip](https://github.com/teamalgoritma/community/files/10246742/Final.Exam.Intro.Data.Science.zip)
<img width="1440" alt="Screenshot 2022-12-16 at 21 56 09" src="https://user-images.githubusercontent.com/88662517/208125437-c56f80c7-5d8b-40e3-8a99-362476c59917.png">

**Comment by yosiaazarya** [comment, marked as answer]
Hal tersebut tampaknya bukanlah masalah melainkan dengan kita melakukan reduksi dengan hanya menggunakan 2 dimensi, maka kita telah mendapatkan ~98% informasi dari data. Hal ini juga berarti bahwa dimensi (PC) 1 menangkap hampir semua informasi yang terkandung pada 5 feature semula pada data.

**Reply by reyhanpurnawan** [reply]
baik terima kasih mas
